config/cmake/scripts/HDF4config.cmake

```diff
@@ -1,22 +1,25 @@
 #############################################################################################
 ### ${CTEST_SCRIPT_ARG} is of the form OPTION=VALUE                                       ###
-### BUILD_GENERATOR required [Unix, VS2017, VS201764, VS2015, VS201564, VS2013, VS201364] ###
-### ctest -S HDFconfig.cmake,BUILD_GENERATOR=VS201764 -C Release -VV -O hdf.log           ###
+### BUILD_GENERATOR required [Unix, VS2022, VS202264, VS2019, VS201964]                   ###
+### ctest -S HDF4config.cmake,BUILD_GENERATOR=VS201764 -C Release -VV -O hdf4.log         ###
 #############################################################################################
 
 cmake_minimum_required (VERSION 3.12)
 ############################################################################
 # Usage:
-#     ctest -S HDFconfig.cmake,OPTION=VALUE -C Release -VV -O test.log
+#     ctest -S HDF4config.cmake,OPTION=VALUE -C Release -VV -O test.log
 # where valid options for OPTION are:
 #     BUILD_GENERATOR - The cmake build generator:
+#            MinGW     * MinGW Makefiles
 #            Unix      * Unix Makefiles
+#            VS2022    * Visual Studio 17 2022
+#            VS202264  * Visual Studio 17 2022
+#            VS2019    * Visual Studio 16 2019
+#            VS201964  * Visual Studio 16 2019
 #            VS2017    * Visual Studio 15 2017
```

Comment:
If BUILD_GENERATOR required [Unix, VS2022, VS202264, VS2019, VS201964] in  line 3, are VS2017 - VS201564 in lines 14 - 17 valid options?